.travis.yml

```diff
@@ -18,7 +18,7 @@ before_install:
   - docker run -p 3307:3306 --name mysql -e MYSQL_ROOT_PASSWORD=rawls-test -e MYSQL_USER=rawls-test -e MYSQL_PASSWORD=rawls-test -e MYSQL_DATABASE=testdb -d mysql/mysql-server:5.6
   
 script:
-  - SBT_OPTS=-J-Xmx3g sbt clean coverage test -Denv.type=test -Dmysql.host=localhost -Dmysql.port=3307 -Ddirectory.url=ldap://localhost:3389 -Ddirectory.password=testtesttest it:compile
```

Comment:
there seem to be issues compiling this when `coverage` is run. i'm temporarily disabling it for debugging purposes.

will figure out a solution _before_ merging.